tox.ini

```diff
@@ -0,0 +1,27 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py36, py37, py38
+
+[testenv]
+deps =
+  absl-py
+  google-api-core
+  google-cloud-spanner >= 1.17.1
+  frozendict
+  portpicker
+allowlist_externals =
+  wget
+  tar
+  chmod
+setenv =
+  VERSION=1.2.0
+  SPANNER_EMULATOR_BINARY_PATH=/tmp/spanner_emulator/emulator_main
+commands =
+    wget https://storage.googleapis.com/cloud-spanner-emulator/releases/{env:VERSION}/cloud-spanner-emulator_linux_amd64-{env:VERSION}.tar.gz -P /tmp/spanner_emulator/
```

Comment:
This requires `/tmp/spanner_emulator` to exist, right? Also, using predictable names in `/tmp` can open up security issues, because any user can create stuff there. How about using a directory inside the git working tree? E.g., create an empty file called `.spanner-emulator/.keep`, add it to git, and add `.spanner-emulator/**` to `.gitignore`.